src/libsystemd/libsystemd.sym

```diff
@@ -303,6 +303,7 @@ global:
         sd_bus_message_close_container;
         sd_bus_message_copy;
         sd_bus_message_read;
+        sd_bus_message_readv;
```

Comment:
This needs to go in the section for v239  at the end.

Reply:
Fix pushed.